client/client.go

```diff
@@ -266,6 +289,7 @@ func MakeConfiguration(file string) (err error) {
 	}
 	fmt.Fprintf(fd, "[api]\n\turl = \"%s\"\n", cfg.API.URL)
 	fmt.Fprintf(fd, "[gpg]\n\thome = \"%s\"\n\tkeyid = \"%s\"\n", cfg.GPG.Home, cfg.GPG.KeyID)
+	fmt.Fprintf(fd, "[TerminalConf]\n\tcolor = off\n")
```

Comment:
Would be good to use lower case section/key names here to keep consistent with the rest of the file